system/src/Grav/Common/Page/Page.php

```diff
@@ -215,17 +215,17 @@ public function translatedLanguages($onlyPublished = false)
      */
     public function untranslatedLanguages($includeUnpublished = false)
     {
-        $filename = substr($this->name, 0, -(strlen($this->extension())));
-        $config = Grav::instance()['config'];
-        $languages = $config->get('system.languages.supported', []);
+        $filename              = substr($this->name, 0, -(strlen($this->extension())));
```

Comment:
Actually I use Reformat in PhpStorm，
that‘s the exact PSR-1&PSR-2 style predefined，
but seems not same as original code，
I will clean them manually for the next update